packages/fastify/src/constants.ts

```diff
@@ -0,0 +1,11 @@
+import { constants } from '@clerk/backend';
+
+export const API_URL = process.env.CLERK_API_URL || 'https://api.clerk.dev';
+export const API_VERSION = process.env.CLERK_API_VERSION || 'v1';
+export const API_KEY = process.env.CLERK_API_KEY || '';
+export const SECRET_KEY = process.env.CLERK_SECRET_KEY || '';
+export const FRONTEND_API = process.env.CLERK_FRONTEND_API || '';
+export const PUBLISHABLE_KEY = process.env.CLERK_PUBLISHABLE_KEY || '';
+export const JWT_KEY = process.env.CLERK_JWT_KEY || '';
```

Comment:
🔧 Similarly, this should be marked as strictly optional.

Reply:
This is not marked anywhere in the codebase as deprecated. :/  I created this ticket to handle it for all packages.

Reply:
how do we mark them as **strictly optional** ?